src/lib/useEthereumTransaction.js

```diff
@@ -47,13 +47,15 @@ export default function useEthereumTransaction(
   const [chainId, setChainId] = useState();
   const [nonce, setNonce] = useState();
   const [gasPrice, setGasPrice] = useState(initialGasPrice); // gwei
+  const [averageGasPrice, setAverageGasPrice] = useState(gasPrice); // gwei
```

Comment:
It was initially unclear to me what this was for, should maybe be renamed `recommendedGasPrice`.